lib/ansible/modules/cloud/ovirt/ovirt_storage_domains.py

```diff
@@ -113,6 +113,21 @@
             - "C(port) - Port of the fibre channel storage server."
             - "C(lun_id) - LUN id."
             - "Note that these parameters are not idempotent."
+    discard_after_delete:
+        description:
+            - "Boolean flag which indicates whether the storage domain should discard after delete."
```

Comment:
Can you please describe what `discard after delete` means?

Reply:
done